_posts/2016-11-29-future.md

```diff
@@ -0,0 +1,206 @@
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+> It's obvious that we all want to see helpful errors!
+
+- [babel/babylon#125](https://github.com/babel/babylon/pull/125): Instead of "Unexpected token" explain that `await` needs to be used in an async function.
+- Do better inference/guessing on user intention to prevent vague errors.
+
+### `babel-init`
+
+Setup Babel more easily for those still fatigued.
+
+- Setup a .babelrc from scratch
+- Reuse the `babel` npm package to default to use `babel-preset-env`
+- Can possibly do a similar thing to `create-react-app` with ejecting to `babel-cli` + `.babelrc`
+
+```sh
+npm install babel
+# prompt and setup preset-env
+./node_modules/.bin/babel-init
+```
+
+### Run [tc39/test262](https://github.com/tc39/test262)
+
+Running official spec tests against Babel to make sure we comply to the spec or at least know when we don't.
+
+### Smoke Tests
+
+[motiz88/babel-smoke-tests](https://github.com/motiz88/babel-smoke-tests).
+
+### Program Analysis
+
+- [Program Slicing](https://en.wikipedia.org/wiki/Program_slicing) with [slice-js](http://slides.com/kentcdodds/faster-javascript#/)
+- Semantic (AST aware) Grepping tool
+
+### `babel --debug`
+
+Print out all info when erroring. Also perf tooling to get more info on how long each plugin takes.
+
+### Syntax plugins:
+
+Opening the gates might lead to an ecosystem of incompatible plugins.
+
+#### Pluggable Parser
+
+There's been a lot of talk about this but not much progress. Acorn/Babylon does already support overriding parser methods (like with JSX/Flow) but were not sure we want to do this just yet. Workaround is to fork it and pass that to Babel.
+
+#### Sweet.js Interop
+
+Previous [issue](https://github.com/babel/babel/issues/568#issuecomment-71716260). Maybe we can just figure out how to have better interop instead.
+
+#### ?
+
+Anything not already brought up here? Please send us a tweet @babeljs, a message on our slack (join at slack.babeljs.io), or create an RFC-like issue in our repo!
+
+## How will it happen? Another Call for Contributors!
+
+Old Issues:
+
+- [Call for Contributors!](https://github.com/babel/babel/issues/1347)
+- [Being prepared for getting hit by a bus](https://github.com/babel/babel/issues/1888)
+
+You might think that as [a project get more widely used](https://npm-stat.com/charts.html?package=babel-core) that more people show up to help out but like most oss projects that aren't backed by a company there is a constant issue with maintaince and sustainability; people get burned out, move on to other cool projects, or get busy with work, life, family, etc.
```

Comment:
"You might think that as [a project gets more widely used](https://npm-stat.com/charts.html?package=babel-core) that more people show up to help out, but like most OSS projects that aren't backed by a company, there is a constant issue with maintenance and sustainability; people get burned out, move on to other cool projects, or get busy with work, life, family, etc."